test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_IncrementalAlterConfigs.cs

```diff
@@ -123,6 +124,28 @@ public void AdminClient_IncrementalAlterConfigs(string bootstrapServers)
                 Assert.Equal(2, describeConfigsResult.Count);
                 Assert.Equal("222", describeConfigsResult[0].Entries["flush.ms"].Value);
                 Assert.Equal("333", describeConfigsResult[1].Entries["flush.ms"].Value);
+
+                if(!TestConsumerGroupProtocol.IsClassic()) {
```

Comment:
This should be just because the docker image for the classic tests is cp-kafka:7.9.0 so maybe add a comment so we remove the condition when we upgrade the classic test cluster